tests/testthat/test-get_source_expressions.R

```diff
@@ -193,6 +193,50 @@ test_that("1- or 2-width octal expressions give the right STR_CONST values", {
   })
 })
 
+test_that("returned data structure is complete", {
+  temp_file <- withr::local_tempfile()
+
+  lines <- c("line_1", "line_2", "line_3")
+  lines_with_attr <- setNames(lines, seq_along(lines))
+  attr(lines_with_attr, "terminal_newline") <- TRUE
+
+  writeLines(lines, con = temp_file)
+  exprs <- get_source_expressions(temp_file)
+  expect_named(exprs, c("expressions", "error", "lines"))
+  expect_length(exprs$expressions, length(lines) + 1L)
+
+  for (i in seq_along(lines)) {
```

Comment:
nit: i think `seq_along(expressions)` is better

Reply:
oh i see... it's more convenient because of the final element. hmm. ok to leave it as is for now.